spec/namespaces/_core.yaml

```diff
@@ -5067,11 +5067,19 @@ components:
     mtermvectors::path.index:
       in: path
       name: index
-      description: Name of the index that contains the documents.
+      description: The name of the index that contains the documents.
```

Comment:
On line 4701, "document" is singular, but it's plural here. Is this intentional?